src/System/Boot.tsx

```diff
@@ -118,10 +119,12 @@ const EnvironmentProvider: FC<
 }
 
 const ThemeProvider: FC<React.PropsWithChildren<unknown>> = ({ children }) => {
+  const forcedTheme = getENV("FORCED_THEME")
+
   const { preferences } = useAppPreferences()
 
   return (
-    <Theme theme={preferences.theme}>
+    <Theme theme={forcedTheme ?? preferences.theme}>
```

Comment:
Do we need to validate forcedTheme or will it fall back to the light theme in case it's not a valid theme?

Reply:
👍